docs/secrets.md

```diff
@@ -183,17 +153,21 @@ use elevated permissions).
 
 ##### Accessing secrets in nuclio functions
 
-The k8s secrets are passed to nuclio functions as environment variables, and their values can be retrieved directly 
+Nuclio functions do not have the MLRun context available to retrieve secret values. Secret values need to be retrieved 
 from the environment variable of the same name. For example, to access the `aws_key` secret in a nuclio function use:
 ```python
 aws_key = os.environ.get("aws_key")
 ```
 
-### Azure Vault
-MLRun can serve secrets from an Azure key Vault. Azure key Vaults support 3 types of entities - `keys`, `secrets` and 
-`certificates`. MLRun only supports accessing `secret` entities.
+#### Azure Vault
+MLRun can serve secrets from an Azure key Vault. 
+
+```{Note}
+Azure key Vaults support 3 types of entities - `keys`, `secrets` and `certificates`. MLRun only supports accessing 
+`secret` entities.
+```
 
-#### Setting up access to Azure key vault
+##### Setting up access to Azure key vault
 To enable this functionality, a secret must first be created in the k8s cluster which contains the Azure key Vault 
```

Comment:
To enable this functionality, a secret must first be created in the k8s cluster **that** contains the Azure key Vault